### DISCUSSION TITLE
Green Software and Health

### DISCUSSION THREAD

**Discussion by MarySHADE** [Green-Software-Foundation/discussions#124] [Introductions]
Hi, I’m Mary (she/her) and I am the coordinator for the [SHADE Research Hub](https://www.kcl.ac.uk/research/shade-research-hub), which explores issues that emerge when AI/digital technologies, health and the environment meet.  I’m based in Cambridge

I have found a wealth of knowledge in the GSF and we regularly feature GSF resources and projects on our  website, in our [newsletter](https://shade-newsletter.beehiiv.com/) and in social media.  

If you’d like to collaborate with SHADE, I’d love to connect with you - either through this forum, or [LinkedIn](https://www.linkedin.com/in/mary-pitt-53107214/) or in person in Cambridge!

**Comment by NAMRATA-WOKE** [comment]
@MarySHADE Thank you for connecting with us! **[Can we find time](https://calendly.com/namratan/introductory-call)** to speak in the next couple of weeks? I'd love to share how we can possibly involve your researchers, and King's College of London's students and faculty in our projects and discussions. 

[Carbon Hack 24](https://hack.greensoftware.foundation/) might be a great way to kick-off our relationship!
